packages/agreement-consumer/.env

```diff
@@ -3,7 +3,7 @@ PORT=3000
 LOG_LEVEL=info
 
 KAFKA_CLIENT_ID="my-app"
```

Comment:
Maybe we should use distinct client id's too? Not sure what they're used for but could help at least for logging/debugging

Reply:
Agree, the client id is just a label, it doesn't affect the behaviour of the application (with, as far as I know, the 
 only exception of quotas), but it should refer to the specific consumer/producer.
As per last call I would suggest:
- group id: `<consumer-name>-group`
- client id: `<consumer-name>`

Reply:
This setting is overridden in the `dev` environment. Anyway, it makes sense to use a proper name in the local environment.